packages/ui-components/src/table/render-row.tsx

```diff
@@ -23,23 +23,23 @@ const Row = styled.div.withConfig({ shouldForwardProp })<RowProps>`
     cursor: ${(props) => (props.onClickRow ? 'pointer' : 'default')};
     display: flex;
 
-    :hover {
+    :active,
+    :focus {
         div {
-            background-color: ${(props) => props.theme.colors.grey1};
+            background-color: ${(props) => props.theme.colors.grey2};
         }
     }
 
-    :active,
-    focused {
+    :hover {
```

Comment:
Does this still work? Wondering because active now is defined before hover that hover might take precedence and active effect being ignored but unsure exactly what would take over here.

Reply:
For context `stylelint` threw a bunch of lint errors for specificity order - i.e. rules should be defined in order of decreasing specificity or something like that. I think here I screwed up and I didn't need to reorder things.  The selectors have the same specificity so last one defined wins, will revert. Good catch!